components/cam/bld/configure

```diff
@@ -1223,10 +1223,10 @@ if (defined $opts{'camiop'}) {
 }
 my $camiop = $cfg_ref->get('camiop') ? "ON" : "OFF";
 
-# The only dycore supported in CAMIOP mode is Eulerian
-if ($camiop eq 'ON' and $dyn_pkg ne 'eul') {
+# The only dycore supported in CAMIOP mode is Spectral Element
```

Comment:
Are we removing support for the Eulerian dycore altogether?  Or is it that the replay option will only work with SE and Eulerian will still work as usual?

Reply:
Just the replay option will work with SE core.  Eulerian core is still the default option for the E3SM SCM for the time being.

Reply:
Note that I'm hoping that we can eventually ditch/delete the Eulerian dycore completely, but we need the SE dycore to be made the default first.